packages/samples/headless-commerce-ssr-remix/app/routes/token.tsx

```diff
@@ -0,0 +1,68 @@
+import {isExpired, parseJwt} from '@/lib/jwt-utils';
+import {LoaderFunctionArgs} from '@remix-run/node';
+import {coveo_accessToken} from '../cookies.server';
+
+export const loader = async ({request}: LoaderFunctionArgs) => {
+  const accessTokenCookie = await coveo_accessToken.parse(
+    request.headers.get('Cookie')
+  );
+
+  if (!isExpired(accessTokenCookie)) {
+    return Response.json({token: accessTokenCookie});
+  }
+
+  const res = await fetch(
+    'https://platform.cloud.coveo.com/rest/search/v2/token?organizationId=searchuisamples',
```

Comment:
🤔 do you remember if this endpoint is available with orgEndpoint?

Reply:
Yep, I tried it and it works with the orgEndpoint! Will use that instead, thx!